gcs/src/test/java/com/google/cloud/hadoop/fs/gcs/GoogleHadoopFileSystemConfigurationTest.java

```diff
@@ -309,7 +310,8 @@ public void testGrpcConfiguration() {
     long grpcWriteTimeout = 20;
     long grpcUploadBufferedRequests = 25;
     long grpcReadSpeedBytesPerSec = 100 * 1024 * 1024;
-    boolean isDirectPathEnabled = false;
+    boolean isDirectPathEnabled = true;
+    boolean isTrafficDirectorEnabled = true;
```

Comment:
nit: do not use `is` prefix for variables, it's used for methods usually

Reply:
Done!